### PR TITLE
chore: add volunteer kalyan-vurugonda

### PR DESCRIPTION
**Pull request by kalyan-vurugonda**

### CODE REVIEW

**Comment by codewithvoid** [comment]
Thanks for volunteering @kalyan-vurugonda . Currently, we don't have category for devops. Created #89 to add that. Meanwhile can you please resolve other comments.

File: data/users/kalyan-vurugonda.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Kalyan Vurugonda",
+    "bio": "DevOps | Open Source Enthusiast",
+    "image": "https://avatars.githubusercontent.com/kalyan-vurugonda",
+    "categories": [
+      {
+        "slug": "DevOps",
```
Comment (codewithvoid) [inline data/users/kalyan-vurugonda.json:7]: lowercase

File: data/users/kalyan-vurugonda.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Kalyan Vurugonda",
+    "bio": "DevOps | Open Source Enthusiast",
```
Comment (codewithvoid) [inline data/users/kalyan-vurugonda.json:3]: Can you pick one of the two?

File: data/users/kalyan-vurugonda.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "image": "https://avatars.githubusercontent.com/kalyan-vurugonda",
+    "categories": [
+      {
+        "slug": "DevOps",
+        "topics": ["Docker", "Kubernetes", "Other Tools"]
```
Comment (codewithvoid) [inline data/users/kalyan-vurugonda.json:8]: You can either mention a tool or drop this.

File: data/users/kalyan-vurugonda.json (hunk 2 above)
Comment (kalyan-vurugonda) [reply data/users/kalyan-vurugonda.json:3]: okay.

File: data/users/kalyan-vurugonda.json (hunk 3 above)
Comment (kalyan-vurugonda) [reply data/users/kalyan-vurugonda.json:8]: okay

**Comment by codewithvoid** [comment]
Thanks Kalyan. This can go in after fixing #92

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**